osgar/lib/pplanner.py

```diff
@@ -0,0 +1,54 @@
+"""
+Simple Path Planner
+"""
+
+
+def find_path(img, start, finish, verbose=False):
+    max_x, max_y = img.shape
+
+    if not (0 <= start[0] < max_x and 0 <=start[1] < max_y):
+        return None
+
+    if not img[start[1]][start[0]]:
```

Comment:
Why not
```suggestion
    if not img[start]:
```
Numpy arrays can be indexed by tuples directly.

Reply:
this style is nicer but the coordinates are swapped